src/main/scala/dxWDL/util/Adjunct.scala

```diff
@@ -0,0 +1,41 @@
+/** dxWDL supports reading some metadata from "adjunct" files - i.e. files in the same directory
+  * as the WDL file.
+**/
+package dxWDL.util
+
+import java.nio.file.Path
+import dxWDL.base.Utils
+
+object Adjuncts {
+  sealed abstract class AdjunctFile
+  final case class Readme(text: String) extends AdjunctFile
+  final case class DeveloperNotes(text: String) extends AdjunctFile
+
+  def findAdjunctFiles(path: Path): Map[String, Vector[AdjunctFile]] = {
+    val file = path.toFile
+    val parentDir = file.getParentFile
+    var wdlName = file.getName
+
+    if (wdlName.endsWith(".wdl")) {
+      wdlName = wdlName.dropRight(4)
+    }
+
+    val readmeRegexp = s"(?i)readme.${wdlName}.(.*)(?:.md)".r
```

Comment:
Perhaps this is intentional, but should the '.'s be escaped? currently the following matches:
```
val test = s"(?i)readme.GLnexus.(.*)(?:.md)".r
"readme#glnexus#stage1#md" match { case test(stage) => println(stage); case _ => println("None")}
```

Reply:
Should these be .+ instead of .*?
The (:?.md) could just be \.md right?

Reply:
Yep, correct on both counts. Fixed now.